### PR TITLE
Feature/interpolants

### PR DESCRIPTION
**Pull request by whaeck**
Adding interpolators and interpolation functions

Also adds python bindings for the interpolation functions and the comparison functions.

### CODE REVIEW

File: python/test/interpolation/Test_scion_interpolation_histogram.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# standard imports
+import unittest
+
+# third party imports
+
+# local imports
+from scion.interpolation import histogram
+
+class Test_scion_interpolation_histogram( unittest.TestCase ) :
+    """Unit test for the horner function."""
```
Comment (BobbiRiedel) [inline python/test/interpolation/Test_scion_interpolation_histogram.py:10]: Is this a unit test for the Horner function or for something else?

File: python/test/interpolation/Test_scion_interpolation_linlin.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# standard imports
+import unittest
+
+# third party imports
+
+# local imports
+from scion.interpolation import linlin
+
+class Test_scion_interpolation_linlin( unittest.TestCase ) :
+    """Unit test for the horner function."""
```
Comment (BobbiRiedel) [inline python/test/interpolation/Test_scion_interpolation_linlin.py:10]: Is this a test for the horner function?

File: python/test/interpolation/Test_scion_interpolation_linlog.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# standard imports
+import unittest
+
+# third party imports
+
+# local imports
+from scion.interpolation import linlog
+
+class Test_scion_interpolation_linlog( unittest.TestCase ) :
+    """Unit test for the horner function."""
```
Comment (BobbiRiedel) [inline python/test/interpolation/Test_scion_interpolation_linlog.py:10]: Is this a test for the horner function?

File: python/test/interpolation/Test_scion_interpolation_loglin.py [hunk 4]
```diff
@@ -0,0 +1,25 @@
+# standard imports
+import unittest
+
+# third party imports
+
+# local imports
+from scion.interpolation import loglin
+
+class Test_scion_interpolation_loglin( unittest.TestCase ) :
+    """Unit test for the horner function."""
```
Comment (BobbiRiedel) [inline python/test/interpolation/Test_scion_interpolation_loglin.py:10]: Is this a test for the horner function?

File: python/test/interpolation/Test_scion_interpolation_loglog.py [hunk 5]
```diff
@@ -0,0 +1,25 @@
+# standard imports
+import unittest
+
+# third party imports
+
+# local imports
+from scion.interpolation import loglog
+
+class Test_scion_interpolation_loglog( unittest.TestCase ) :
+    """Unit test for the horner function."""
```
Comment (BobbiRiedel) [inline python/test/interpolation/Test_scion_interpolation_loglog.py:10]: Is this a test for the horner function?

File: python/test/math/Test_scion_math_compare.py [hunk 6]
```diff
@@ -0,0 +1,31 @@
+# standard imports
+import unittest
+
+# third party imports
+
+# local imports
+from scion.math import is_close, is_close_to_zero
+
+class Test_scion_math_horner( unittest.TestCase ) :
+    """Unit test for the horner function."""
```
Comment (BobbiRiedel) [inline python/test/math/Test_scion_math_compare.py:10]: Is this a test for the horner function?

**Review by BobbiRiedel — COMMENTED** [review]
A few comments in the unit testing need to be updated to reflect the interpolation functors being tested. Real small changes.

File: python/test/interpolation/Test_scion_interpolation_histogram.py (hunk 1 above)
Comment (whaeck) [reply python/test/interpolation/Test_scion_interpolation_histogram.py:10]: That's a typo, probably a copy/paste error on my part. I fixed that.

**Comment by whaeck** [comment]
All documentation changes are in #53

**Review by BobbiRiedel — APPROVED** [review]

**Pull request merged**